app/models/agents/fleep_agent.rb

```diff
@@ -0,0 +1,53 @@
+module Agents
+  class FleepAgent < Agent
+    include FormConfigurable
+
+    cannot_be_scheduled!
+    cannot_create_events!
+    no_bulk_receive!
+
+    description <<-MD
+      The Fleep Agent sends messages to a Fleep conversation
+
+      To authenticate you need to set the `fleep_conversation_webhook_url`, you can get one by configuring a generic webhook like described here: https://fleep.io/blog/integrations/webhooks/
```

Comment:
I can't remember if our Markdown parser auto converts URLs to links, if it does this is fine, otherwise it would be nice to have the URL as a clickable link.